Tests with Iris 3.16 and xxhash 4.0.0 fail

**Describe the bug**

Since Iris 3.16 came out last week, pyaerocom tests are failing.

Please provide a clear and concise description of what the bug is.
 - Pyaerocom version: latest
 - Computing platform: PyPi (not conda)
 - Error message (if applicable):
```
>           result = xxh64_hexdigest(str(parts))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
E           TypeError: Strings must be encoded before hashing

/home/runner/work/pyaerocom/pyaerocom/.tox/py/lib/python3.13/site-packages/iris/common/metadata.py:107: TypeError

```

**To Reproduce**
Run test-pipeline on github

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,6 +26,8 @@ dependencies = [
     # and minimum versions in the tox config at the bottom of this file!
     "aerovaldb>=0.5.2, < 0.6.0",
     "scitools-iris>=3.11.0",
+    # xxhash and iris incompatibility issue: https://github.com/SciTools/iris/issues/7251
+    "xxhash<=4.0.0",
     "xarray>=2022.12.0",
     "cartopy>=0.21.1",
     "matplotlib>=3.7.1",
@@ -235,6 +237,8 @@ deps =
     # Should be incremented when changing dependencies above.
     aerovaldb ==0.5.2; python_version < "3.11"
     scitools-iris ==3.11.0; python_version < "3.11"
+    # xxhash and iris incompatibility issue: https://github.com/SciTools/iris/issues/7251
+    xxhash <= 4.0.0
     cartopy ==0.21.1; python_version < "3.11"
     matplotlib ==3.7.1; python_version < "3.11"
     scipy ==1.10.1; python_version < "3.11"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,7 @@ dependencies = [
     "aerovaldb>=0.5.2, < 0.6.0",
     "scitools-iris>=3.11.0",
     # xxhash and iris incompatibility issue: https://github.com/SciTools/iris/issues/7251
-    "xxhash<=4.0.0",
+    "xxhash<4.0.0",
     "xarray>=2022.12.0",
     "cartopy>=0.21.1",
     "matplotlib>=3.7.1",
@@ -238,7 +238,7 @@ deps =
     aerovaldb ==0.5.2; python_version < "3.11"
     scitools-iris ==3.11.0; python_version < "3.11"
     # xxhash and iris incompatibility issue: https://github.com/SciTools/iris/issues/7251
-    xxhash <= 4.0.0
+    xxhash < 4.0.0
     cartopy ==0.21.1; python_version < "3.11"
     matplotlib ==3.7.1; python_version < "3.11"
     scipy ==1.10.1; python_version < "3.11"

